app/imports/components/mainLayout.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import Header from '/imports/components/header.jsx';
+import Footer from '/imports/components/footer.jsx';
+
+export const MainLayout = ({header, content, footer}) => (
```

Comment:
this is the same as
`export const MainLayout = function({header, content, footer}) {`

the vars are in brackets cuz of react-komposer i think… not sure